cypress/integration/transfer/reorder-with-buttons.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Reorder items in the selected list
+
+  Background:
+    Given reordering of items is enabled
+
+# I created two scenarios for up and down reordering because I wanted to incorporate a check for 'is first' or 'is last' and it seemed overly ambiguous to write this in a single scenario. Or, is it enough that I have included the 'Disable move buttons' scenarios at the bottom of this file, so that doesn't need to be handled in the basic scenarios at the top?
+
+  Scenario: Reorder items upwards using buttons
+    Given a selected list with one or more items
+    And a highlighted item in the selected list
+    And the highlighted item is not the first item in the list
+    When the 'move up' button is clicked
+    Then the highlighted item moves up one place in the selected list
+
+  Scenario: Reorder items downwards using buttons
+    Given a selected list with one or more items
+    And a highlighted item in the selected list
+    And the highlighted item is not the last item in the list
+    When the 'move down' button is clicked
+    Then the highlighted item moves down one place in the selected list
+
+  Scenario: Disable reorder buttons when no items highlighted
+    Given a selected list with one or more items
+    When no items are highlighted in the list
+    Then the 'move up' and 'move down' reorder buttons are disabled
+
+  Scenario: Disable 'move up' button when highlighted item is first
+    Given a selected list with at least one item
+    When a highlighted item is first in the list
+    Then the 'move up' button is disabled
```

Comment:
`When` should be `Given` or `And`.
As above, I'd change the `Then` step here in the same way.
The button is not disabled, it's press event is simply ignored.

Reply:
I've updated this, but I just want to check we are on the same page.
> The button is not disabled, it's press event is simply ignored.
The button should be disabled (as in, use `<Button disabled>`. Are you saying that this information should not be included in a feature file?

Reply:
Oh, no. I though you were talking about the "page up" keyboard buttons. In that case you scenarios are actually correct! Sorry 'bout that

Reply:
Oh, oops! No problem, that's good to know. I'll revert the changes I made for that.